src/webgpu/api/operation/memory_sync/buffer/buffer_sync_test.ts

```diff
@@ -255,4 +255,12 @@ export class BufferSyncTest extends GPUTest {
     bufferData[0] = expectedValue;
     this.expectContents(buffer, bufferData);
   }
+
+  verifyDataMultipleValidValues(buffer: GPUBuffer, expectedValue1: number, expectedValue2: number) {
```

Comment:
nit: this could take a `sequence<number>`

Reply:
same reason